raphtory/src/search/mod.rs

```diff
@@ -277,14 +277,13 @@ mod test_index {
             let binding = tempfile::TempDir::new().unwrap();
             let path = binding.path();
 
-            graph.cache(path).unwrap();
 
             graph
                 .add_node(1, "Ozai", [("prop", 1)], Some("fire_nation"))
                 .unwrap();
 
             // This also tests if already existing index is replaced by new index
-            graph.write_updates().unwrap();
+            // graph.write_updates().unwrap();
```

Comment:
remove instead of comment

Reply:
Replaced it with a TODO since search is currently fully broken and we need to come back to it later.